code-challenges.test.js

```diff
@@ -14,54 +14,158 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
+describe("fibSequence",()=>{
+  
+  const fibLength1 = 6
+  const fibLength2 = 10
+
+  it("returns an array that contains the numbers of the fibonacci sequence", ()=> {
+
+    expect(fibSequence(fibLength1)).toEqual ([1, 1, 2, 3, 5, 8])
+    expect(fibSequence(fibLength2)).toEqual ([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    
+  })
+})
+// FAIL  ./code-challenges.test.js
+//   fibSequence
+//     ✕ returns an array that contains the numbers of the fibonacci sequence (1 ms)
+
+//   ● fibSequence › returns an array that contains the numbers of the fibonacci sequence
+
+//     TypeError: "fibSequence" is not a function
+
 // Expected output: [1, 1, 2, 3, 5, 8]
 
-const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
+//Pseudo:
+//Input:number greater than 2
+//Output: array
+//we will create a function called fibSequence that will first check if the number is less than 3. If not it will return an eror. Then we will create a function fibNum that will take in an array of 1 and 1 because 1 and 1 will be the first two numbers of the fib sequence. next a for loop to calculate the numbers of the fib sequence and store them in an array. Then console log the given fibLength numbers and we should expect to see [ 1, 1, 2, 3, 5, 8 ] and [  1,  1,  2,  3,  5,  8, 13, 21, 34, 55].
+
+const fibSequence = (num) => {
+  if (num < 3) {
+    return ("something went wrong")
+  }
+
+const fibNum = ([1,1]); {
```

Comment:
Nice! This is a great start! You have some extra syntax in here that isn't necessary, like the () around the array of fibNum. I love the idea of a for loop and you can definitely do this within one function, I am not quite sure what you mean by create a function fibNum in your pseudocode. You could also be more descriptive with where you will start and stop the for loop, as well as how you will reference the last 2 numbers in the function.